en/docs/install-and-setup/setup/deployment/kubernetes-deployment-patterns.md

```diff
@@ -59,16 +59,19 @@ However, the following set of integration artifacts are stateful and requires co
 
-<img src="{{base_path}}/assets/img/integrate/k8s_deployment/k8s-muliple-workers.png">
+!!! Tip
+    - See [Coordination configurations]({{base_path}}/install-and-setup/setup/deployment/configuring-helm-charts/#coordination-configurations) for instructions on configuring coordination across multiple Micro Integrator instances using the Helm charts.
+    - If you dynamically change the state of a Message Processor or Inbound Endpoint using the Management API or Integration Control Plane, you must share the registry across Micro Integrator instances to persist the state when new nodes join the cluster. Refer to [Registry synchronization]({{base_path}}/install-and-setup/setup/deployment/deploying-wso2-mi/#registry-synchronization-sharing) for more information. Registry synchronization is an optional setup and is not required for basic coordination.
 
-### Load balancing
+<img src="{{base_path}}/assets/img/integrate/k8s_deployment/k8s_coordination.png">
 
-Because stateful artifacts cannot be deployed in multiple containers/pods, it is also not possible to distribute the workload for a stateful artifact when there is high traffic. However, it is possible to reduce the workload on a single worker node or pod by isolating individual stateful artifacts (or groups of selected artifacts) in individual pods.
+!!! Note
+    As long as you maintain a single artifact deployment for each of these artifacts, coordination is not required. You can arrange your cluster in the following manner to ensure that the same task is not deployed in multiple containers/pods in the cluster.
 
-In the example shown above, you can assume that `recurringOrder_Task` and `schedulOrder_Task` are highly-utilized scheduled tasks in your deployment. By deploying them in two separate pods (<b>POD 2</b> and <b>POD 3</b>, you have optimized resource utilization.
+    <img src="{{base_path}}/assets/img/integrate/k8s_deployment/k8s-muliple-workers.png" width="80%">
 
 ### High availability
 
-Because stateful artifacts (that require coordination) are deployed in one container/pod in one worker node, if the node fails or if the pod fails, the pod will be spawned again in one of the running working nodes. This avoids single point of failure. However, there will be a downtime until the pod deployment becomes active again.
+When stateful artifacts are deployed with coordination enabled across multiple replicas, only one instance actively executes the task at a time, while others remain passive. If the active instance (pod or node) becomes unavailable due to a failure, one of the passive replicas will automatically take over execution. This ensures high availability and prevents downtime, as the failover happens seamlessly without service interruption.
```

Comment:
1. Someone might misunderstand this with LB level active/passive.
2. If there are multiple stateful artifacts it gets distributed across multiple nodes.

Reply:
Fixed with 15fa15e